packages/mobile/package.json

```diff
@@ -164,11 +164,11 @@
     "@babel/plugin-proposal-decorators": "^7.12.1",
     "@celo/flake-tracker": "0.0.1-dev",
     "@celo/typescript": "0.0.1",
-    "@graphql-codegen/add": "^1.10.0",
-    "@graphql-codegen/cli": "^1.10.0",
-    "@graphql-codegen/fragment-matcher": "1.10.0",
-    "@graphql-codegen/typescript": "1.10.0",
-    "@graphql-codegen/typescript-operations": "1.10.0",
+    "@graphql-codegen/add": "^2.0.2",
+    "@graphql-codegen/cli": "^1.21.3",
+    "@graphql-codegen/fragment-matcher": "2.0.1",
+    "@graphql-codegen/typescript": "1.21.1",
+    "@graphql-codegen/typescript-operations": "1.17.15",
```

Comment:
Is it really necessary to update this now?
I just did a quick test and it breaks `yarn run build:gen-graphql-types`
The generated types are slightly different and not directlty compatible with our usage, specifically with `IntrospectionFragmentMatcher`.

Reply:
Yes unfortunately this was the minimal set of changes. But I encountered other plugin errors so closing this PR in favor of https://github.com/celo-org/wallet/pull/234 where I just use the resolutions to pin the version to 5.0.5.